pulser/_seq_drawer.py

```diff
@@ -280,12 +287,15 @@ def phase_str(phi):
         # Terminate the last open regions
         if target_regions:
             target_regions[-1].append(t[-1])
-        for start, targets, end in target_regions:
+        for start, targets, end in target_regions:  # type: ignore
```

Comment:
```
pulser\_seq_drawer.py:291: error: Incompatible types in assignment (expression has type "object", variable has type "List[Any]")
```
And it is not possible to type hint directly between `for` and `in`.

Reply:
I think you just have to cast `targets` to `list` on the next line.

Reply:
```
pulser\_seq_drawer.py:291: error: Incompatible types in assignment (expression has type "object", variable has type "List[Any]")
pulser\_seq_drawer.py:293: error: Redundant cast to "List[Any]"
```

Reply:
Ahh, it might be because `targets` was already being used before. Try using `targets_` here instead (make sure you replace every instance inside the `for` loop) and see if it helps